resources/monitoring/profile-production.yaml

```diff
@@ -0,0 +1,27 @@
+prometheus:
+  prometheusSpec:
+    resources:
+      limits:
+        cpu: 600m
```

Comment:
CPU requests and limits are also not fitting, see: https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-production.yaml.tpl

Reply:
Fixed thanks